lib/generator.js

```diff
@@ -224,7 +224,8 @@ const generateRest = (comments, customTypes, usedTypes) => {
   for (const comment of comments) {
     let line;
     if (comment.name === 'Example' || comment.name === 'Result') {
-      const code = comment.comment.trimLeft();
+      let code = comment.comment.trimLeft();
+      if (!code.endsWith(';')) code += ';';
```

Comment:
Would this correctly work for
```js
Example:
callAsyncFunction((a, b) => {
  if (a) {  
    console.log(b)
  }
})
```
?

Reply:
Yes, but you'll need to add `;` after `console.log(b)`.

Reply:
Could you add a test for that, please?